Framework/SINQ/src/PoldiTruncateData.cpp

```diff
@@ -68,7 +68,7 @@ void PoldiTruncateData::setTimeBinWidthFromWorkspace(
         "Workspace does not contain any data. Aborting.");
   }
 
-  const MantidVec &xData = workspace->readX(0);
+  auto &xData = workspace->x(0);
```

Comment:
`const` is not required. `auto` already provides `const HistogramData::HistogramX`.